assets/model_monitoring/components/src/shared_utilities/momo_exceptions.py

```diff
@@ -4,7 +4,15 @@
 """This file define exceptions used in Model Monitoring pipeline."""
 
 
-class DataNotFoundError(Exception):
+class UserError(Exception):
+    """Exception raised when it is user error."""
+
+    def __init__(self, message):
+        """Initialize a DataNotFoundError."""
+        super().__init__(f'UserError: {message}')
+
+
+class DataNotFoundError(UserError):
```

Comment:
Do you have any job with this error shown? we want to check how is the exception thrown and logged

Reply:
After checking the job, the user error does not help for telemetry. Removed this code change.